src/timeAgo.ts

```diff
@@ -86,27 +91,32 @@ export const timeAgo = (
 	epoch: number,
 	options?: {
 		extended?: boolean;
+		daysUntilAbsolute?: number;
 	},
 ): false | string => {
 	const then = new Date(epoch);
 	const now = new Date();
 	const extended = options?.extended;
+	const daysUntilAbsolute = options?.daysUntilAbsolute ?? 7;
 
 	if (!isValidDate(then)) {
 		return false;
 	}
 
 	const secondsAgo = Math.floor((now.getTime() - then.getTime()) / 1000);
+	const veryClose = secondsAgo < 15;
```

Comment:
Is it within `5` or `15`? The tests use `5`.

Reply:
Fixed the tests 👍